pkg/flare/flare.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util"
 	httputils "github.com/DataDog/datadog-agent/pkg/util/http"
 	"github.com/DataDog/datadog-agent/pkg/version"
+	"github.com/pkg/errors"
```

Comment:
invalid import

Reply:
This import was added by my Go plugin (for `errors.New`) and the code compiles fine on GitLab.
I remove it and it is now replaced by "errors". Strange.

Reply:
yeah actually I realize now that `github.com/pkg/errors` does exist (see https://github.com/pkg/errors), and we seem to be using it in a few packages (`pkg/ebpf` for example).

Anyway, let's stick to `errors` here